jbmc/regression/jbmc-cover/generics/test.desc

```diff
@@ -1,9 +1,6 @@
 CORE
 AbstractTest.class
---cover location --function AbstractTest.getFromAbstract
```

Comment:
Since the test makes sense without the `cover` option, it can be moved out of `jbmc-cover` (as in commit message).

Reply:
Fixed.